src/components/phase/card/PhaseCard.tsx

```diff
@@ -0,0 +1,134 @@
+import classNames from 'classnames'
+import { useCallback, useEffect, useRef, useState } from 'react'
+import { useMediaType } from '../../../hook/useMediaType'
+import type { PhaseDTO, Duration, Task } from '../../../data/phase/dto.type'
+import ClearIcon from '@mui/icons-material/Clear'
+import LoupeIcon from '@mui/icons-material/Loupe'
+import moment from 'moment'
+import Link from 'next/link'
+
+type DropDownProps = Pick<PhaseDTO, 'phaseName' | 'tasks'> & {
+  refObj: React.RefObject<HTMLDivElement> | null
+  onClose: () => void
+  phaseDuration: Duration
+}
+
+export type PhaseCardProps = PhaseDTO
+
+export const DropDown: React.FC<DropDownProps> = ({
+  tasks,
+  phaseName,
+  onClose,
+  refObj,
+  phaseDuration
+}): JSX.Element => {
+  const from = moment(phaseDuration.from).utc().format('MMM. Do')
+  const to = moment(phaseDuration.to).utc().format('MMM. Do')
+
+  return (
+    <div className="okp4-nemeton-web-phasecard-dropdown-container" ref={refObj}>
+      <div className="okp4-nemeton-web-phasecard-dropdown-tasks">
+        <ClearIcon className="okp4-nemeton-web-phasecard-dropdown-clear-icon" onClick={onClose} />
+        <div className="okp4-nemeton-web-phasecard-dropdown-duration">
+          <p>
+            Duration: {moment(phaseDuration.to).diff(moment(phaseDuration.from), 'weeks')} weeks
+          </p>
+          <p>{`From ${from} to ${to}`}</p>
+        </div>
+
+        <div className="okp4-nemeton-web-phasecard-dropdown-task">
+          {tasks.map(({ group: subject, taskName, points }, index: number, array) => {
```

Comment:
Why do you rename the `group` property?

Reply:
Good catch, this is something I missed to fix while I moved the file. I will remove the namespace